src/components/init-modal/screens/api-keys/index.js

```diff
@@ -109,19 +109,18 @@ export default ( { onSetupStatus } ) => {
 								disabled: true,
 								label: __( 'Select service provider', 'newspack-newsletters' ),
 							},
-							{ value: 'manual', label: __( 'Manual', 'newspack-newsletters' ) },
-							{ value: 'mailchimp', label: __( 'Mailchimp', 'newspack-newsletters' ) },
+							{ value: 'mailchimp', label: 'Mailchimp' },
 							{
 								value: 'constant_contact',
-								label: __( 'Constant Contact', 'newspack-newsletters' ),
+								label: 'Constant Contact',
```

Comment:
Same question: why no translation?

Reply:
Constant Contact is a brand so it shouldn't be translated.